tools/rna_tools/cofold/cofold.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="cofold" name="Cofold" version="0.0.1">
```

Comment:
The tool version needs also to be bumped as soon as you change the tool. I prefer 2.0.4.0. Last digit is if you change the tool without changing the requirement.